modules/tinymce/src/plugins/link/test/ts/browser/QuickLinkTest.ts

```diff
@@ -156,4 +159,26 @@ describe('browser.tinymce.plugins.link.QuickLinkTest', () => {
       'a:contains("Word")': 1
     });
   });
+
+  it('TINY-8057: Checking that mceLink command can open Quicklink and dialog', async () => {
+    const editor = hook.editor();
+    editor.setContent('');
+    editor.execCommand('mcelink', false, true);
+    await TinyUiActions.pWaitForDialog(editor);
+    TinyUiActions.closeDialog(editor);
+    editor.execCommand('mcelink');
+    // Waiting for Quicklink dialog as in pOpenQuickLink
+    await Waiter.pWait(100);
```

Comment:
Using `Waiter.pWait` is normally a pretty big red flag, as in many cases it'll lead to flaky tests. It should only be used when there's nothing to wait for generally. So in this case we do have something to wait for and that's for the context toolbar UI to render. For example:
```ts
await TinyUiActions.pWaitForPopup(editor, '.tox-pop .tox-toolbar');
```
and then as that's more descriptive, you probably don't need the comment above 🤷 

- [ ] Don't use `Waiter.pWait` to wait for the quicklink context toolbar to show

Reply:
Done, thanks Lee